### DISCUSSION TITLE
Hardware for bonding fast lines (100Mbps+)

### DISCUSSION THREAD

**Discussion by fasternet1** [Ysurac/openmptcprouter#2014] [Q&A]
Hello Yannick & Community,

I was wondering what kind of hardware (performance wise) do I need to be able to bond 2 WANs while each having 100Mbps or more on download.

Should a support for AES instruction set (all recent AMD CPUs, Intel CPUs i5 and above probably?) do it or does it then need to have for example at least 8 cores to fully utilize 2x100Mbps? What about 2x200Mbps?

Any feedback is greatly appreciated.

**Comment by ioogithub** [comment]
Hi, did you ever find any additional information on this? You asked good questions that I would also like to know. What did you end up using?

**Comment by Ysurac** [comment]
There is already several issues to ask what is best hardware.
It's always x86_64 with AES-NI and the best core speed (better to have 1 fast core than 8 slow cores)
